.github/workflows/update_gh_pages.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow updates GitHub Pages on the repo it is run from.
+
+name: Update GitHub Pages
+
+on:
+  workflow_dispatch:  # Manually trigger. Colon is required.
+
+permissions:
+  contents: write  # For checkout and push.
+
+jobs:
+  update-github-pages:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        ref: master
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
```

Comment:
Any reason not to use v18?  It will be supported longest.

Reply:
Nah, I just copied the publish workflow. Will update.

Reply:
Not all of the plugins are compatible with node 18. I think it might be the eslint config. Keeping it as node 16 for now but will file an issue for node 18 compatibility.

Reply:
SGTM.